### ISSUE TITLE
Training of Wide-Resnet for Coco datasets

### ISSUE THREAD

**Issue by alceubissoto** [Faruk-Ahmed/syst-gen#3]
Hi again Faruk!

When you were training the wideresnet28-4 with the coco colors dataset, did you manage to have a high (>90%) accuracy on the training set? And did you use dropout? 
I can't go past 68% acc during training, even though the performance on the iid test set is around 80% acc (which is a little strange...). Do you have any suggestion?

Best regards!

**Comment by alceubissoto** [comment]
Also, may I ask for the ideal hyperparameters (λ and the number of epochs over which to linearly ramp up λ) for the coco datasets? :D

**Comment by Faruk-Ahmed** [comment]
I don't recall anything funny going on with the training accuracies (I believe they were higher than tests), so I'm not sure why you're experiencing this. It's been a while, so I might have forgotten something there. I did not use dropout. I've described some details in Appendix B.2, maybe it'll help.

Regarding best hyper-parameters, I think I probably used the best values in the README, which are (wc = lambda, u = ramp):
`python main.py -dataset colour -method pgi -wc 100.0 -u 200`
`python main.py -dataset places -sample normal -method pgi -wc 50.0 -u 200`
